### PR TITLE
add logzero package

### PR DESCRIPTION
**Pull request by carlodri**

### CODE REVIEW

File: recipes/logzero/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - colorama
```
Comment (synapticarbors) [inline recipes/logzero/meta.yaml]: `colorama` appears to be a windows-only requirement. If this is the case, please add the proper selector:

```
- colorama  # [win]
```

If that's added then you have to remove the `noarch: python`

File: recipes/logzero/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  summary: 'Python logging made easy'
+  description: |
+    Robust and effective logging for Python 2 and 3.
+  doc_url: https://logzero.readthedocs.io/
+  dev_url: https://github.com/metachris/logzero
+
+extra:
+  recipe-maintainers:
+    - carlodri
+    - metachris
```
Comment (synapticarbors) [inline recipes/logzero/meta.yaml]: @metachris are you ok being added as a co-maintainer on the `logzero` recipe on conda-forge?

https://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role

Comment (carlodri) [reply recipes/logzero/meta.yaml]: @metachris [has been warned](https://github.com/metachris/logzero/issues/67).... 😉

File: recipes/logzero/meta.yaml (hunk 1 above)
Comment (carlodri) [reply recipes/logzero/meta.yaml]: thanks, you're right. Fixing in a minute.

File: recipes/logzero/meta.yaml (hunk 2 above)
Comment (metachris) [reply recipes/logzero/meta.yaml]: Yes! Thanks for the support.

**Comment by metachris** [comment]
Thanks!

**Pull request merged**